### DISCUSSION TITLE
Can we use TAP reporter and c8 coverage at same run?

### DISCUSSION THREAD

**Discussion by yukha-dw** [avajs/ava#3302] [Q&A] [answered]
As title said, or do we have to run the test twice as ava will change its output to support [ tap reporter](https://github.com/avajs/ava/blob/main/docs/05-command-line.md#tap-reporter)?

`c8 --check-coverage --100 ava --tap | tap-nyan`

**Comment by novemberborn** [comment, marked as answer]
Yes, you can. The reporter is just what's printed, `c8` instruments Node.js itself and picks up coverage files that are written to disk.
